frontend/src/routes/SmallBusinessSupport/SmallBusinessSupport.js

```diff
@@ -24,8 +26,9 @@ const SmallBusinessSupport = () => {
   const { sections } = sectionsLayout
   return (
     <Container sx={{ mt: 3 }}>
+      <SectionNavs sections={sections} />
```

Comment:
To make the stickiness of the scrollspy easier to test, add in a placeholder for the content that will eventually be at the top of the page:

```suggestion
      <Typography variant="h1" gutterBottom>
        Adelaide
      </Typography>
      <Box sx={{height: 400, backgroundColor: 'lightGray'}} mb={4} />
      <SectionNavs sections={sections} />
```